docs/PLUGINS.md

```diff
@@ -218,6 +218,12 @@ Add the Configuration as Code plugin as a test dependency in your pom.xml:
 </dependency>
 ```
 
+#### Configuration-as-code.yml
+
+##### **Location of configuration-as-code.yml:**
+
+The `configuration-as-code.yml` file should be located within the test resources directory (`src/test/resources`) of your project repository. Specifically, it should be placed in the same directory where your test classes reside.
```

Comment:
I think this should go in the next section as it hasn't been mentioned about a configuration file yet, you could probably add an example file with the file path? content looks good though, almost there thanks.

Reply:
@timja I have added more information please do let me know if anything is wrong or If I need to add something more